docs-source/usersguide/application.rst

```diff
@@ -584,14 +584,27 @@ File                                 Parameters
 
 As a convenience, the file :file:`amber14-all.xml` can be used as a shortcut to include
 :file:`amber14/protein.ff14SB.xml`, :file:`amber14/DNA.OL15.xml`, :file:`amber14/RNA.OL3.xml`,
-and :file:`amber14/lipid17.xml`.  In most cases you can simply include that file,
-plus one of the water models.
+and :file:`amber14/lipid17.xml`.  In most cases, you can simply include that file,
+plus one of the water models, such as :file:`amber14/tip4pew.xml` for the TIP4P-Ew
+water model\ :cite:`Horn2004`:
+::
+
+    forcefield = ForceField('amber14-all.xml', 'amber14/tip4pew.xml')
```

Comment:
Most users will have 3-site water molecules in their topology, so it's probably better to give a 3-site model as the example.  I suggest TIP3P-FB.

Reply:
Thanks! I can add a reminder that a step through `app.Modeller` will be needed if four-site waters are needed.